src/i18n/it/README.md

```diff
@@ -0,0 +1,56 @@
+
+## 🔄️ Vocaboli che non hanno bisogno di traduzione
+
+> 💡 Queste parole sono considerate nomi propri data la loro origine Inglese :
+
+| Vocabolo          | Definizione | Uso         |
+|-------------------|-------------|-------------|
+| Runtime           | Ambiente d'esecuzione / Tempo di esecuzione | Usato come "runtime di JavaScript" o "runtime edge" |
+| Framework         | Parola comunemente usata nel codice per specificare un software o un pacchetto che fa uso di una sintassi specifica al suo ambiente (es: `React`, `Vue`, `Svelte`, ecc...) | Utilizzato in riferimento a vari ambienti che Astro integra e supporta
+| Build             | Nel contesto di un lingaggio di programmazione che richiede un passaggio di compilazione e/o traduzione (tra due diversi linguaggi) | Questa parola deve essere usata quando si parla della fase di compilazione tramite il comando `astro build` usato per produrre un risultato compatibile con i browser e ottimizzato da Astro
+| Frontend          | Tutto quell'insieme di pacchetti e programmazione usati per costruire l'interfaccia utente direttamente visibile nel browser | Comunemente usato per distinguere il codice relativo alla logica dell'interfaccia utente dal codice che viene eseguito sul server e non è visibile all'utente |
+| Backend           | L'insieme del codice che veine usato per definire la struttura e il funzionamento del server | Vocabolo usato in contrapposizione a "Frontend" per distinguere codice e logica scritti per il browser da quelli scritti per il server |
+| Markdown, Astro, JavaScript, TypeScript, React, Vue, Svelte, Lit, Solid | Nomi di pacchetti o linguaggi di programmazione sempre scritti con la maiuscola | Utilizzati ovunque nella documentazione e sempre scritti con la maiuscola |
+| API (Application Programming Interface) | Interfaccia di programmazione dell'applicazione ovvero l'insieme dei metodi, classi e funzioni che definiscono il funzionamento e l'uso di una libreria o un framework | Abbreviazione usata in riferimento all'architettura e interfaccia di programmazione di Astro e altri software che potrebbero essere integrati con il framework |
+
+## 📚 Traduzioni comuni
+
+Alcuni vocaboli hanno una correlazione diretta o quasi con l'Italiano e vanno usate sempre allo stesso modo per mantenere una certa coerenza all'interno della documentazione:
+
+| Inglese                      | Italiano                                           |
+|------------------------------|----------------------------------------------------|
+| SSR / Server-side rendering  | SSR / Rendering lato server                        |
+| Client-side                  | Lato browser / Lato client                         |
+| Template                     | Template / Modello                                 |
+| CLI / Command line interface | CLI / Interfacia da riga di comando                |
+| Routing                      | Indirizzamento                                     |
```

Comment:
I would consider "routing" as a non-translatable word, together with "router". On the other hand "rotta" as "route" is widely used also in Italian.
What do you think?

Reply:
I can't really think of a graceful translation. "Indirizzamento" looked to me as the best one I could find and honestly I wasn't sure whether to put it among the non translatable words or not.

Maybe someone else in the server can give their opinion about it too.

Reply:
I think the translation "indirizzamento" is widely used in the context of IP routing (e.g. in physical routers, or in operating system IP tables), less in web application routing.

How would you refer to the "router" then? "Indirizzatore"? (tooooooo baaad!!! :laughing: ).
And "route"? "Indirizzo" makes me think of a full URL, but here we're talking just about the path.

My suggestion, based on quick search of random articles in Italian containing "expressjs routing" or "laravel routing", would be:
- routing -> not translated
- router -> not translated
- route -> both "route" and "rotta" are acceptable.

E.g., though not authoritative, I found this good translation related to Laravel. 
https://www.html.it/pag/54739/gestire-il-routing-con-laravel/

Definitely, we could ask some opinions on Discord.

Reply:
I totally agree :v: